### PR TITLE
Allow PIMT to be re-run in case of failure

### PR DESCRIPTION
**Pull request by nre-ableton**
While working on another feature, I realized that there was a bit of a shortcoming in https://github.com/Ableton/ansible-role-jenkins-jcasc/pull/261.

### CODE REVIEW

**Review by mst-ableton — APPROVED** [review]
code lgtm, but I'm confused by the commit message of eaa1f62347e5f08df97a612bd06e8b19efcdeea9. As far as I can tell, the code is never converted to `shell` in this PR.

**Comment by nre-ableton** [comment]
> code lgtm, but I'm confused by the commit message of [eaa1f62](https://github.com/Ableton/ansible-role-jenkins-jcasc/commit/eaa1f62347e5f08df97a612bd06e8b19efcdeea9). As far as I can tell, the code is never converted to `shell` in this PR.

Erm, yes, it will still be in a future commit, but I decided to split off that commit into another branch for further testing. Sorry about the confusion. :sweat_smile:

File: tasks/install-plugins.yml [hunk 1]
```diff
@@ -19,23 +19,45 @@
...
+      retries: "{{ jenkins_download_retries }}"
+      delay: "{{ jenkins_download_delay }}"
+      register: plugin_installation_result
+      until: plugin_installation_result is succeeded
+  rescue:
+    # Because we only run PIMT when the template file changed, we can end up in a bad
+    # situation if PIMT fails for other reasons (for instance, a network glitch). This
+    # will leave the Jenkins controller in a partially deployed state, and when this role
+    # is run again, we need to ensure that PIMT will also be re-run. To do so, we alter
+    # the plugins.yaml file, which will ensure that the template task wil change it.
```
Comment (ala-ableton) [inline tasks/install-plugins.yml:49]:
```suggestion
    # the plugins.yaml file, which will ensure that the template task will change it.
```
@nre-ableton FYI

Comment (nre-ableton) [reply tasks/install-plugins.yml:49]: I added a cleanup commit to https://github.com/Ableton/ansible-role-jenkins-jcasc/pull/263.

Comment (ala-ableton) [reply tasks/install-plugins.yml:49]: Thanks!

**Pull request merged**